authors/apps/articles/tests/test_article_likes_and_dislikes.py

```diff
@@ -0,0 +1,113 @@
+from rest_framework import status
+from django.urls import reverse
+from .base_setup import Base
+
+
+class ArticleLikeTests(Base):
+    def setUp(self):
+        super().setUp()
+        response = self.client.post(self.article_url, self.article_data,
+                                    format="json", **self.headers)
+        self.article_id = response.data['id']
+        self.likes_dislikes_url = reverse(
+            'articles:likeArticles', kwargs={'pk': self.article_id})
+        self.like = {'like': True}
+        self.dislike = {'like': False}
+
+    def tearDown(self):
+        super().tearDown()
+
+    def test_can_like_an_article(self):
+        """
+        Tests that a user can like an article
+        """
+        response = self.client.post(self.likes_dislikes_url,
+                                    data=self.like,
+                                    format="json",
+                                    **self.headers
+                                    )
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+    def test_cannot_like_an_article_more_than_once(self):
+        """
+        Tests that a user cannot like/dislike an article more than once
+        """
+        self.client.post(self.likes_dislikes_url,
+                         format="json",
+                         data=self.like,
+                         **self.headers
+                         )
+        response = self.client.post(self.likes_dislikes_url,
+                                    format="json",
+                                    data=self.like,
+                                    **self.headers
+                                    )
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def test_cannot_like_non_existing_article(self):
+        """
+        Tests that a user cannot like an article which does not exist
+        """
+        response = self.client.post(
+            reverse('articles:likeArticles', kwargs={'pk': -1}),
+            format="json",
+            data=self.like,
+            **self.headers
+        )
+        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+
+    def test_can_dislike_an_article(self):
+        """
+        Tests that a user can dislike an article
+        """
+        response = self.client.post(self.likes_dislikes_url,
+                                    data=self.dislike,
+                                    format="json",
+                                    **self.headers
+                                    )
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
```

Comment:
@meshack-mbuvi When a user is liking and disliking an article, the status code should be 200 Ok.